### ISSUE TITLE
fix(e2e): assign nonce manually for parallel transactions

### ISSUE THREAD

**Issue by manishbista28** [icon-project/icon-bridge#539]
## Overview
When a lot of parallel transfer calls are made to bsc chain with the same source address, then nonce returned by ethclient will not have been updated to reflect all parallel requests. Even if the transaction is nth (n > 1) transaction in the batch, eth client will still return nonce used by the first transaction.
Because the nonce value has to be different, we should not rely on nonce retrieved from ethclient but instead find a way to assign the value ourselves.

# Proposed Solution
The solution will not fully rely on eth client but itself use a tracker to increment nonce value of a source address.

## Steps to Reproduce
1. Make a lot of parallel requests from the same source address to different destination address
2. Destination address can be of the same or different chains (easier to replicate for same chain i.e bsc -> bsc txn)
3. Observe result return by transfer call

## Expected Behavior
Chain specific behavior should be handled by chain API. In this case, bsc chain api should handle how to track nonce.

**Comment by bbist** [comment]
One potential issue with assigning nonce manually is that if the txn with older nonce is not picked up for some reason (low gas price), all other transactions will remain in pending state. But for e2e test, with enough gasPrice, this shouldn't be an issue, I suppose.

**Comment by manishbista28** [comment]
I agree with the potential issue. Assigning nonce manually does not seem like a reasonable solution to the original problem as there can be multitude of other ways transaction may not get submitted.
